README.md

```diff
@@ -21,7 +21,7 @@ If you would like to access your dashboard at http://localhost, simply define th
 
 ```diff
 {
-+   "default": "/absolute/path/to/valet-dashboard",
```

Comment:
The current path already matches the project name, as if you had simply `git clone`d the project. Did you clone yours differently or rename it perhaps?